specification/maintenance/resource-manager/Microsoft.Maintenance/preview/2022-11-01-preview/Maintenance.json

```diff
@@ -0,0 +1,2564 @@
+            "description": "Definition of a MaintenanceOverrideProperties",
+            "x-ms-client-flatten": true
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "MaintenanceConfigurationProperties": {
+      "description": "Properties for maintenance configuration",
+      "type": "object",
+      "properties": {
+        "namespace": {
+          "description": "Gets or sets namespace of the resource",
+          "type": "string"
+        },
+        "extensionProperties": {
+          "description": "Gets or sets extensionProperties of the maintenanceConfiguration",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "maintenanceScope": {
+          "description": "Gets or sets maintenanceScope of the configuration",
+          "enum": [
+            "Host",
+            "Resource",
+            "OSImage",
+            "Extension",
+            "InGuestPatch",
+            "SQLDB",
+            "SQLManagedInstance"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "MaintenanceScope",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Host",
+                "description": "This maintenance scope controls installation of azure platform updates i.e. services on physical nodes hosting customer VMs."
+              },
+              {
+                "value": "Resource",
+                "description": "This maintenance scope controls the default update maintenance of the Azure Resource"
+              },
+              {
+                "value": "OSImage",
+                "description": "This maintenance scope controls os image installation on VM/VMSS"
+              },
+              {
+                "value": "Extension",
+                "description": "This maintenance scope controls extension installation on VM/VMSS"
+              },
+              {
+                "value": "InGuestPatch",
+                "description": "This maintenance scope controls installation of windows and linux packages on VM/VMSS"
+              },
+              {
+                "value": "SQLDB",
+                "description": "This maintenance scope controls installation of SQL server platform updates."
+              },
+              {
+                "value": "SQLManagedInstance",
+                "description": "This maintenance scope controls installation of SQL managed instance platform update."
+              }
+            ]
+          }
+        },
+        "maintenanceWindow": {
+          "$ref": "#/definitions/MaintenanceWindow",
+          "description": "Definition of a MaintenanceWindow",
+          "x-ms-client-flatten": true
+        },
+        "overrides": {
+          "$ref": "#/definitions/MaintenanceConfigurationOverrides",
+          "description": "Definition of a MaintenanceConfigurationOverrides",
```

Comment:
>MaintenanceConfigurationOverrides [](http://example.com/codeflow?start=42&length=33)

Can you explain what this is for in the description? Simply stating the model name is not beneficial for customers to understand what it needs to be used for